SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -96,7 +96,15 @@ void USpatialActorChannel::DeleteEntityIfAuthoritative()
 	// If we have authority and aren't trying to delete a critical entity, delete it
 	if (bHasAuthority && !IsSingletonEntity())
 	{
-		Sender->SendDeleteEntityRequest(EntityId);
+		// Temporary hack to delay the delete entity request if tearing off.
```

Comment:
Can you add a jira ticket and reference it here please?

Reply:
Done so 👍